packages/block/src/types.ts

```diff
@@ -116,7 +117,9 @@ export interface BlockData {
    * Header data for the block
    */
   header?: HeaderData
-  transactions?: Array<TxData | AccessListEIP2930TxData | FeeMarketEIP1559TxData>
+  transactions?: Array<
+    TxData | AccessListEIP2930TxData | FeeMarketEIP1559TxData | BlobEIP4844TxData
```

Comment:
What we could also do here is, export this type (so all types combined) from our `tx` library, and then import that type here..? (As a slight refactor)

Reply:
Agreed but I'm actually doing this WIP PR as part of a larger tx interfaces refactoring: https://github.com/ethereumjs/ethereumjs-monorepo/pull/2767

Reply:
Allright :+1: